docs/ossm/cert-manager/README.md

```diff
@@ -0,0 +1,200 @@
+     --wait \
+     --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+     --set "volumeMounts[0].name=root-ca" \
+     --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+     --set "volumes[0].name=root-ca" \
+     --set "volumes[0].secret.secretName=istio-root-ca" \
+     --set "app.istio.namespace=istio-system"
+   ```
+
+   Note: If your controlplane namespace is not `istio-system`, you will need to update `app.istio.namespace` to match your controlplane namespace.
+
+2. Install `Istio` controlplane in the `istio-system` namespace.
+
+   ```sh
+   oc apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: default
+   spec:
+     version: v1.23.0
+     namespace: istio-system
+     updateStrategy:
+       type: InPlace
+     values:
+       global:
+         caAddress: cert-manager-istio-csr.cert-manager.svc:443
+       pilot:
+         env:
+           ENABLE_CA_SERVER: "false"
+         volumeMounts:
+           - mountPath: /tmp/var/run/secrets/istiod/tls
+             name: istio-csr-dns-cert
+             readOnly: true
+   EOF
+   ```
+
+3. [Verify your deployment](#verify) is configured correctly.
+
+## RevisionBased Strategy
+
+For the `RevisionBased` strategy, you need to specify all the istio revisions in your [istio-csr deployment](https://github.com/cert-manager/istio-csr/tree/main/deploy/charts/istio-csr#appistiorevisions0--string).
+
+1. Install [istio-csr](https://cert-manager.io/docs/usage/istio-csr).
+
+   ```sh
+   helm repo add jetstack https://charts.jetstack.io --force-update
+
+   helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+     --install \
+     --namespace cert-manager \
+     --wait \
+     --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+     --set "volumeMounts[0].name=root-ca" \
+     --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+     --set "volumes[0].name=root-ca" \
+     --set "volumes[0].secret.secretName=istio-root-ca" \
+     --set "app.istio.namespace=istio-system" \
+     --set "app.istio.revisions={default-v1-23-0}"
+   ```
+
+   Note: If your controlplane namespace is not `istio-system`, you will need to update `app.istio.namespace` to match your controlplane namespace.
+
+2. Install `Istio` controlplane in the `istio-system` namespace.
+
+   ```sh
+   oc apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: default
+   spec:
+     version: v1.23.0
+     namespace: istio-system
+     updateStrategy:
+       type: RevisionBased
+     values:
+       global:
+         caAddress: cert-manager-istio-csr.cert-manager.svc:443
+       pilot:
+         env:
+           ENABLE_CA_SERVER: "false"
+         volumeMounts:
+           - mountPath: /tmp/var/run/secrets/istiod/tls
+             name: istio-csr-dns-cert
+             readOnly: true
+   EOF
+   ```
+
+3. [Verify your deployment](#verify) is configured correctly.
+
+### Verify
+
```

Comment:
For clarity: these verification steps apply to both the `InPlace` strategy and the `RevisionBased` strategy?

Reply:
Yes but there's one step of `verify` that varies between the two